rasa_sdk/knowledge_base/actions.py

```diff
@@ -149,20 +150,38 @@ def _query_objects(
         Returns: list of slots
         """
         object_type = tracker.get_slot(SLOT_OBJECT_TYPE)
-        object_attributes = self.knowledge_base.get_attributes_of_object(object_type)
+        if utils.is_coroutine_action(self.knowledge_base.get_attributes_of_object):
+            object_attributes = await self.knowledge_base.get_attributes_of_object(
```

Comment:
should be `async` in the interface

Reply:
I guess, we should also update `InMemoryKnowledgeBase` and use `async`.